ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/util/ForkChoiceUtil.java

```diff
@@ -173,10 +173,12 @@ public static void on_tick(Store store, UnsignedLong time) {
   /**
    * @param store
    * @param block
+   * @param st
    * @see
    *     <a>https://github.com/ethereum/eth2.0-specs/blob/v0.8.1/specs/core/0_fork-choice.md#on_block</a>
    */
-  public static void on_block(Store store, BeaconBlock block) throws StateTransitionException {
+  public static void on_block(Store store, BeaconBlock block, StateTransition st)
```

Comment:
Seems like a good fix passing in StateTransition instead of instantiating it each time

Reply:
+1